draft-cdn-control-header.md

```diff
@@ -75,11 +75,13 @@ shown here.
 
 # The CDN-Control Response Header Field
 
-The CDN-Control Response Header Field allows origin servers to control the behaviour of gateway caches interposed between them and clients, separately from other caches that might handle the response.
+The CDN-Control response header field allows origin servers to control the behaviour of gateway caches interposed between them and clients, separately from other caches that might handle the response.
 
-It is a Dictionary Structured Header {{!I-D.ietf-httpbis-header-structure}}, whose members can be any directive registered in the HTTP Cache Directive Registry <https://www.iana.org/assignments/http-cache-directives/http-cache-directives.xhtml>.
+It is a Dictionary Structured Header {{!I-D.ietf-httpbis-header-structure}}, whose members are cache-directives defined in {{!RFC7234}}. In practice, they can be any directive registered in the HTTP Cache Directive Registry <https://www.iana.org/assignments/http-cache-directives/http-cache-directives.xhtml>.
```

Comment:
Right I think we can break this section down to 3 parts. 1. syntax, which I think Dictionary Structured Header should be enough. 2. semantic: the well defined ones are from the registry. 3. We need something like https://tools.ietf.org/html/rfc7234#section-5.2.3. Since Cache-Control itself is extendable, CDN-control should allow the same. The words in it current state sound like that is not allowed.

Reply:
Could you perhaps back this one change out so we can merge this PR, and then we can work on that?

Reply:
done